tests/test_core.py

```diff
@@ -8377,7 +8377,6 @@ def test_undefined_main(self):
 
   def test_fpic_static(self):
     self.emcc_args.append('-fPIC')
-    self.emcc_args.remove('-Werror')
```

Comment:
why is this changed?

Reply:
Because we no long print a warning when `fPIC` is used.